components/kyma-environment-broker/common/orchestration/dto.go

```diff
@@ -103,9 +103,10 @@ type ParallelStrategySpec struct {
 
 // StrategySpec is the strategy part common for all orchestration trigger/status API
 type StrategySpec struct {
-	Type     StrategyType         `json:"type"`
-	Schedule ScheduleType         `json:"schedule,omitempty"`
-	Parallel ParallelStrategySpec `json:"parallel,omitempty"`
+	Type          StrategyType         `json:"type"`
+	Schedule      ScheduleType         `json:"schedule,omitempty"`
```

Comment:
I'm thinking of the following (partially backwards incompatible) API change: 

- `MaintenanceWindow` should be a bool parameter, it can be omitted (default false).
- `Schedule` can be of value `immediate` (to keep backwards compatibility), `now` , or a specific parsable timestamp format.
- When neither the above parameters are specified, or only schedule=immediate/now is specified, the current default behavior is executed, i.e. all operations are executed immediately by the parallel strategy
- When schedule=maintenanceWindow is specified (using older version of KCP CLI), the request is rejected with 400 Bad request
- When MaintenanceWindow is set to true, the initial maintenance windows are resolved for each operation. If `Schedule` is set to a valid time, operations are scheduled after that timestamp. The reschedule logic in parallel strategy for maintenance window operations will also need to be readjusted. 
- When MaintenanceWindow is set to false and `Schedule` is set to a valid time, the maintenance window begin/end are not resolved, but the parallel strategy will enqueue all operations with .`AddAfter(Schedule)` 

The KCP CLI should be adjusted to accommodate these param changes: the `--schedule` should accept `immediate`, `now` or a valid timestamp parable into `time.Time`.